gpkit/nomial_data.py

```diff
@@ -0,0 +1,128 @@
+import numpy as np
+
+from collections import defaultdict
+
+from functools import reduce as functools_reduce
+from operator import add
+
+from .small_classes import HashVector, Quantity
+
+from .small_scripts import mag
+
+
+class NomialData(object):
+
```

Comment:
should the kwarg `signomials` be renamed `nomials` to make it clear it can be anything?